packages/hooks/README.md

```diff
@@ -0,0 +1,22 @@
+# `@sewing-kit/hooks`
+
+Hooks are a pluggable way of extending or configuring components of a Workspace (Packages, WebApps, or Services) that we want to perform tasks on.  Core hooks are part of sewing-kit and along with custom hooks they comprise the configuration that defines how these tasks are performed.
+
+There are different types of hooks which define the set of steps to be completed as part of any task. Waterfall or Series hook types.
+
+There are also context objects for each of the Project types (Service, Package, or WebApp)
+
+With the options, hooks and context, a task can run.
+
+[Plugins](/packages/plugins/README.md) can be defined at the workspace or project level.  
+
+Both Workspace and Plugin context provide access to the Plugin API that allows you to create a step for a task
+
+## Installation
+
+```
+yarn add @sewing-kit/hooks --dev
+```
+
+## Usage
```

Comment:
I think we should dive into some of the capabilities enabled by `SeriesHook` and `WaterfallHook` hook classes with examples of `.hook` and `.run` being used.

Reply:
Since your review, I added to the above.  Will also find some examples.